extensions/amp-ad/0.1/test/test-amp-ad.js

```diff
@@ -134,6 +123,7 @@ describes.realWin('Ad loader', {amp: true}, env => {
             .withArgs('amp-ad-network-zort-impl')
             .returns(Promise.reject(new Error('I failed!')));
         ampAd = new AmpAd(ampAdElement);
+        sandbox.stub(ampAd.user(), 'error');
```

Comment:
why is this needed?

Reply:
Gets around allowConsoleError which given the error is thrown within a promise chain is a real pain to indicate expected appearance.  This seemed the easiest way to make it work (can revisit later).